docs/administration-guide/01-managing-databases.md

```diff
@@ -57,26 +57,39 @@ To add a database, you'll need its connection information.
+### Database Sync and Analysis
 
-When Metabase connects to your database, it tries to decipher the field types in your tables based on each field's name. Metabase also takes a sample of each table to look for URL's, json, encoded strings, etc. If a field is classified wrong, you can always manually edit it from the **Metadata** tab in the Admin Panel.
+By default, Metabase performs a lightweight hourly sync of your database, and a nightly deeper analysis of the fields in your tables to power some of Metabase's features, like filter widgets.
 
-### Metadata Syncing
+If you'd like to change these default settings, find on click on your database in the Databases section of the Admin Panel, and turn on the toggle at the bottom of the form that says "This is a large database, so let me choose when Metabase syncs and scans." (This is an option that used to be called "Enable in-depth analysis.")
 
-Metabase maintains it's own information about the various tables and fields in each Database that is added to aid in querying.  This information is generally updated once each night to look for changes to the database such as new tables, but if you'd like to sync your database manually at any time:
+![Large database toggle](images/large-db-toggle.png)
 
-NOTE: Metabase does NOT copy any data from your database, it only maintains lists of the tables and columns.
+Save your changes, and you'll see a new tab at the top of the form called "Scheduling." Click on that, and you'll see options to change when and how often Metabase syncs and scans.
 
-1. Go to the Admin Panel.
+#### Database syncing
 
-2. Select **Databases** from the navigation menu.
-![adminbar](images/AdminBar.png)
+Metabase maintains its own information about the various tables and fields in each database that is added to aid in querying. By default, Metabase performs this lightweight sync hourly to look for changes to the database such as new tables or fields. Metabase does *not* copy any data from your database. It only maintains lists of the tables and columns.
 
-3. Click on the database you would like to sync.
-![databaselist](images/DatabaseList.png)
+Syncing can be set to hourly, or daily at a specific time. Syncing can't be turned off completely, otherwise Metabase wouldn't work.
 
-4. Click on the **Sync** button on the right of the screen.
-![databaseconnection](images/DatabaseConnection.png)
+If you'd like to sync your database manually at any time, click on it from the Databases list in the admin panel and click on the Sync button on the right side of the screen:
+
+![Database connection](images/DatabaseConnection.png)
+
+#### Scanning for field values
+
+When Metabase first connects to your database, it tries to guess the types of the fields in your tables based on each field's name. Metabase also takes a sample of each table to look for URLs, JSON, encoded strings, etc. If a field is classified wrong, you can always manually edit it from the **Metadata** tab in the Admin Panel.
```

Comment:
`it tries to guess the types of the fields in your tables based on each fields's name` isn't exactly 100% correct here. It's more a combination of the field's underlying type in the DB, the data, and the field name that influences the types.